infrastructure/helm-chart/charts/tools/charts/akhq/templates/configmap.yaml

```diff
@@ -17,11 +22,13 @@ data:
           properties:
             isolation.level: read_committed
       connections:
-        core:
+      {{- range .Values.kafka.namespaces }}
+        {{ . }}:
           properties:
-            bootstrap.servers: ${KAFKA_BROKERS}
+            bootstrap.servers: {{ $bootstrapServers }}.{{ . }}:{{ $kafkaPort }}
```

Comment:
Even if multiple brokers exist in the namespace, this is the URL of the headless service which selects all of the brokers, so it will not be a problem.